docs/section15/Rule15-10.md

```diff
@@ -0,0 +1,28 @@
+# Transformers - Rule 15-10
+**Rule ID:** Rule 15-10  
+**Rule Description:** Transformer capacity ratio reported in Proposed RMR equals User RMR  
```

Comment:
I wonder if this is the right rule. Shouldn't this instead be that the regulated aspects of the transformers (capacity, efficiency, phase) are the same in U-RMR and P-RMR? I am not sure we need to require peak_load equivalency, because in theory there are any differences in loads between U-RMR and P-RMR.